crates/chain/src/chain_graph.rs

```diff
@@ -465,7 +465,7 @@ where
 #[must_use]
 pub struct ChangeSet<P> {
     pub chain: sparse_chain::ChangeSet<P>,
-    pub graph: tx_graph::Additions,
+    pub graph: tx_graph::Additions<()>,
```

Comment:
```suggestion
    pub graph: tx_graph::Additions,
```

Reply:
It doesn't work.

Reply:
Put a default for the `A` in `TxGraph` and it will :)